content/membership/members.toml

```diff
@@ -73,6 +73,11 @@ name = "infra.run"
 website = "https://infra.run"
 logo = "infra.run.svg"
 
+[[silver]]
+name = "Liberbyte"
+website = "https://www.liberbyte.com/"
+logo = "liberbyte.png"
```

Comment:
as you already will have expected: I would have hoped for an svg but oh well here we are :)